YvesIraguha/yves-iraguha-portfolio
# use correct contet

#### What does this PR do?
- Replace dummy content with the correct content. 
#### Description of Task to be completed?
- Replace the text on experience, and education with correct content
- Use correct images on project preview
- Use the correct title and icon in the title. 
#### How should this be manually tested?
- Clone the repo and checkout the branch `correct-content-resume-page`
- Run `yarn` and `yarn start` to start the application 
- Check on the `resume` page, you should see the attached screenshot. 

#### Screenshots (if appropriate)

![resume](https://user-images.githubusercontent.com/34691070/66474536-cce18500-ea91-11e9-982b-0679910932a9.png)
